LoggingKata.Test/TacoParserTests.cs

```diff
@@ -8,22 +8,32 @@ public class TacoParserTests
         [Fact]
         public void ShouldDoSomething()
         {
-            // TODO: Complete Something, if anything
+            //
         }
 
         [Theory]
-        [InlineData("Example")]
+        [InlineData("0, 0, test")]

```

Comment:
Consider adding some more `InlineData` to test different cases, such as "20.5, -20.5, Test", "-20.5, 20.5, Test", etc.